src/main/java/frc/robot/subsystems/ClimberSubsystem.java

```diff
@@ -77,7 +101,10 @@ public double getRightHeight() {
   }
 
   public void setHeight(double height) {
-    left.getPIDController().setReference(height, ControlType.kSmartMotion);
-    // right.getPIDController().setReference(height, ControlType.kSmartMotion);
+    if (Constants.ClimberConstants.minimumHeight < height
+        && height < Constants.ClimberConstants.maximumHeight) {
+      left.getPIDController().setReference(height, ControlType.kPosition);
+      right.getPIDController().setReference(height, ControlType.kPosition);
+    }
```

Comment:
just generally lots of stuff from tuning still in here that i don't know if we want to clean out for clarity